CONTRIBUTING.md

```diff
@@ -99,3 +112,7 @@ commits pulled from the LLVM community repository
 
 *Other names and brands may be claimed as the property of others.
 
+**For any commit not related to DPC++, but rather to LLVM in general, it is
```

Comment:
Please move it up as the very first bullet in "Development" section. This is important to know before even starting making a patch.

Reply:
Done.